src/internal.c

```diff
@@ -29371,7 +29374,8 @@ int SendClientKeyExchange(WOLFSSL* ssl)
                 case ecc_diffie_hellman_kea:
                 {
                     /* place size of public key in buffer */
-                    *args->encSecret = (byte)args->encSz;
+                    if (args->encSecret != NULL) 
```

Comment:
Should we error out if it is somehow NULL?

Reply:
Good idea. Preferred ret value for this case?

Reply:
For these sorts of things that should "never happen," BAD_STATE_E is one option.

Reply:
Done